oap-server/server-receiver-plugin/skywalking-trace-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/trace/provider/TraceServiceModuleConfig.java

```diff
@@ -51,6 +51,9 @@
     @Setter
     @Getter
     private UninstrumentedGatewaysConfig uninstrumentedGatewaysConfig;
+    @Setter
+    @Getter
+    private TraceSampleRateWatcher traceSampleRateWatcher;
```

Comment:
Why config has reference to watcher? This is strange. Watcher should be a part of `TraceSegmentSampler`

Reply:
Config should only be changed through the initialization stage. Nothing more.

Reply:
i understand this mean this config have which watchers will modify this config‘s properties？for example,same with uninstrumentedGatewaysConfig be added before

Reply:
OK, ignore this. Continue on others.